src/util/log.js

```diff
@@ -0,0 +1,126 @@
+import { getEnv } from "./util";
+
+function getDefaultLogObject() {
+  return {
+    level: "info",
+    tags: ["screener-front-end"],
+    message: {
+      systemURL: window.location.href,
+    },
+  };
+}
+//write to audit log
+// @param level, expect string
+// @param tags, expect array, e.g. ['etc']
+// @param message, expect object, e.g. { "questionId": "123"}
+export function writeToLog(level, tags, message) {
+  const confidentialBackendURL = getEnv("VUE_APP_CONF_API_URL");
+  if (!confidentialBackendURL) {
+    console.log("confidential backend URL is not set.");
```

Comment:
Always appreciate when a config is optional! 

If I understand correctly, if `VUE_APP_CONF_API_URL` is unconfigured, no logging will occur. 

Just added some clarification:

```suggestion
    console.log("audit log skipped; confidential backend URL is not set");
```